src/Ansatz/Chain.jl

```diff
@@ -502,17 +502,17 @@ function evolve_1site!(qtn::Chain, gate::Dense)
     contract!(TensorNetwork(qtn), contracting_index)
 end
 
-function evolve_2site!(qtn::Chain, gate::Dense; threshold, maxdim)
+function evolve_2site!(qtn::Chain, gate::Dense; threshold, maxdim, iscanonical = false)
```

Comment:
I think it would be better if we used a method name, not a kwarg.

Reply:
Are you sure? Both functions use quite a lot of the same function internals. I think that in the long run, we could have a field inside `Chain` that specifies the canonical form. For me it makes sense to have that as a `kwarg`.

Reply:
Okay, let's discuss the name offline then.